libvast/src/format/csv.cpp

```diff
@@ -410,16 +410,23 @@ caf::error reader::read_impl(size_t max_events, size_t max_slice_size,
                              consumer& callback) {
   VAST_ASSERT(max_events > 0);
   VAST_ASSERT(max_slice_size > 0);
+  bool timeout = false;
+  auto next_line = [&] {
+    if (!builder_ || builder_->rows() == 0)
+      lines_->next();
+    else
+      timeout = lines_->next_timeout(
```

Comment:
Same here, don't we need to return if `timeout` is `true`?

Reply:
Fixed both, I accidentally dropped that when rebasing. Thanks for noticing.